jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -423,7 +423,7 @@ public boolean handle()
                             ServletHandler.MappedServlet mappedServlet = _servletContextRequest._mappedServlet;
```

Comment:
Can't we put the `try/finally` of calling the listeners inside this block?

Reply:
We can.

I thought it was neater with the boolean, because then only 1 try/finally and it stays close to the notification of the combinedListener. 
But I don't mind will change it to this.